### PR TITLE
BONK support & MODE env

### PR DESCRIPTION
**Pull request by DonDuala**
No urgent changes. Running locally

### CODE REVIEW

File: src/scalper.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (brittcyr) [inline src/scalper.ts:1]: can these be more targeted than just disabling entire files.

File: src/config.ts [hunk 2]
```diff
@@ -21,22 +24,33 @@ export const DELTA_OFFSET = new Map<string, number>([
...
+  ['BONK', parseFloat(bonk_vars[3]) > 0 ? parseFloat(bonk_vars[3]) : 2.58],
 ]); // Corresponds to 80% CI by default
 
+export const MODE = new Map<string, number>([
+  ['BTC', parseFloat(btc_vars[4]) > 0 ? parseFloat(btc_vars[4]) : 0],
+  ['ETH', parseFloat(eth_vars[4]) > 0 ? parseFloat(eth_vars[4]) : 0],
+  ['SOL', parseFloat(sol_vars[4]) > 0 ? parseFloat(sol_vars[4]) : 0],
+  ['MNGO', parseFloat(mngo_vars[4]) > 0 ? parseFloat(mngo_vars[4]) : 0],
+  ['BONK', parseFloat(bonk_vars[4]) > 0 ? parseFloat(bonk_vars[4]) : 0],
+]); // 0 - Normal 1 - Gamma+Back 2- Back Only
```
Comment (brittcyr) [inline src/config.ts:52]: Make the mode an enum

File: src/scalper_utils.ts [hunk 3]
```diff
@@ -445,3 +446,17 @@ export async function getJupiterPrice(
   const midPrice = (buyPrice + sellPrice) / 2;
   return midPrice;
 }
+
+export function setPriorityFee(
+  tx: Transaction
+) {
+  const modifyComputeUnits = ComputeBudgetProgram.setComputeUnitLimit({
```
Comment (brittcyr) [inline src/scalper_utils.ts:453]: Do we need this many CU? Setting this too big will get you deprioritized

File: src/scalper.ts (hunk 1 above)
Comment (DonDuala) [reply src/scalper.ts:1]: will plan to address linting later

File: src/config.ts (hunk 2 above)
Comment (DonDuala) [reply src/config.ts:52]: getting errors, todo added

File: src/scalper_utils.ts (hunk 3 above)
Comment (DonDuala) [reply src/scalper_utils.ts:453]: Just referencing https://solanacookbook.com/references/basic-transactions.html#how-to-change-compute-budget-fee-priority-for-a-transaction

Todo to refine this logic, we should try to make it dynamic based on demand per market. ie. BONK was having issues this week but SOL fine

**Pull request merged**